### DISCUSSION TITLE
Github Apps - revoking partial permissions

### DISCUSSION THREAD

**Discussion by MichaelKetting** [community/community#38382] [Other Feature Feedback, Questions, & Ideas]
Hello, 

due to a recent change in the permissions of one of the Github Apps I'm using (https://github.com/atlassian/github-for-jira/issues/1728), I came to realize that there isn't a way to deny some permissions for a Github App. 

For instance, the Jira App for Github needs Write for Contents just to create branches. I don't need this feature and want to deny just this one permissions. Is there a way to do this that I haven't discovered yet?

Many thanks, Michael

**Comment by doublerebel** [comment]
Related: #31605, #15733
